src/Core/src/Exception/Http/HttpExceptionTrait.php

```diff
@@ -68,6 +69,13 @@ public function __construct(ResponseInterface $response)
             }
         }
 
+        $logger->error($message, [
```

Comment:
I'm not a big fan of loging in an exception's constructor :/

Reply:
That is fair.. But where else can I use it?

Reply:
Oh, In the Response class of course =)